app/app/result/page.jsx

```diff
@@ -0,0 +1,102 @@
+'use client';
+import '../globals.css';
+import clsx from 'clsx';
+import TableContainer from '@mui/material/TableContainer';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import { useEffect, useState } from 'react';
+
+const ResultsPage = () => {
+  const [results, setResults] = useState([]);
+
+  useEffect(() => {
+    const getResults = async () => {
+      try {
+        const res = await fetch('/api/results');
+        const data = await res.json();
+        setResults(data);
+      } catch (error) {
+        console.error('Error:', error.message);
```

Comment:
We discussed on Sunday that it is a must to show an error to user if something went wrong. Have you created a task regarding this?